spec/models/user_spec.rb

```diff
@@ -644,48 +577,15 @@
     end
   end
 
-  describe '#password_updated_at' do
-    let(:user) { users(:affiliate_admin) }
-
-    it 'is set when the user is created' do
-      expect(user.password_updated_at).to_not be_nil
-    end
-
-    it 'is set when the password is updated' do
-      expect { user.update(password: "new1234!") }.
-        to change{ user.password_updated_at }
-    end
+  describe '#self.from_omniauth' do
+    subject(:from_omniauth) { OmniAuth.config.mock_auth[:login_dot_gov] }
 
-    it 'is not set when other attributes are updated' do
-      expect { user.update(contact_name: 'Kermit the Frog') }.
-        to_not change{ user.password_updated_at }
+    it 'sets the users email' do
```

Comment:
It looks like this would test the scenario where a new user is signing up from Login.gov. The `.from_omniauth` specs should be broken into two contexts for each of the potential scenarios (`first_or_create`):
```ruby
describe '.from_omniauth' do
  context 'when the user is new' do
    ...
  end

  context 'when the user already has a Search.gov account' do
     ...
  end
end
```